src/test/java/mt/edu/um/cf2/jgribx/TestRunner.java

```diff
@@ -4,11 +4,14 @@
 import org.junit.runner.Result;
 import org.junit.runner.notification.Failure;
 
-public class TestRunner {
-    public static void main(String[] args) {
+public class TestRunner
```

Comment:
Do we need this class? Gradle should handle running tests, so I'm guessing we can chop this but wanted to verify.

Reply:
That is a weird file, don't need it at all, so it's gone!\